docs/api.rst

```diff
@@ -4716,6 +4789,14 @@ Dealing with History Events
    :reqjson string notes: This is a description of the event entry in plain text explaining what is being done. This is supposed to be shown to the user.
    :reqjson string event_subtype: Optional. An optional subtype for the entry. Possible event types can be seen in HistoryEventSubType enum.
    :reqjson string counterparty: Optional. An identifier for a potential counterparty of the event entry. For a send it's the target. For a receive it's the sender. For bridged transfer it's the bridge's network identifier. For a protocol interaction it's the protocol.
+   :reqjson int block_number: This is the index of the block the event is contained in.
+   :reqjson int validator_index: This is the index of the validator.
```

Comment:
can you check the errors? I bleieve the text need updates. For example if there is a constraint that fails like the validator index doesn't exists, I believe there is one like that

Reply:
The errors seem correct